scripts/importDpsConstants.ts

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env npx tsx
+import { spawnSync } from 'node:child_process'
+import fs from 'node:fs'
+import path from 'node:path'
+
+import type { IncidentReportingApi, Constant, TypeConstant } from '../server/data/incidentReportingApi'
+
+type ConstantsMethod = keyof IncidentReportingApi['constants']
+interface Template {
+  method: ConstantsMethod
+  enumName: string
+  documentation?: string
+}
+const templates: Template[] = [
+  { method: 'types', enumName: 'Type' },
+  { method: 'statuses', enumName: 'Status' },
+  { method: 'informationSources', enumName: 'InformationSource' },
+  { method: 'staffInvolvementRoles', enumName: 'StaffInvolvementRole' },
+  { method: 'prisonerInvolvementRoles', enumName: 'PrisonerInvolvementRole' },
+  { method: 'prisonerInvolvementOutcomes', enumName: 'PrisonerInvolvementOutcome' },
+  { method: 'correctionRequestReasons', enumName: 'CorrectionRequestReason' },
+  {
+    method: 'errorCodes',
+    enumName: 'ErrorCode',
+    documentation: 'Unique codes to discriminate errors returned from the incident reporting api',
+  },
+]
+
+function printHelp(): never {
+  const help = `
+Imports constants from incident reporting api downloaded as JSON files.
+Usage:
+  ./scripts/importDpsConstants.ts [type] [file path]
+
+Where [type] is one of ${templates.map(template => template.method).join(', ')}
+`.trim()
+  process.stderr.write(`${help}\n`)
+  process.exit(1)
+}
+
+const [, , type, filePath] = process.argv
```

Comment:
> would need to be nested inside main because it captures outer variables
the use of `templates`? Well, that's a constant, it could go inside `main() {}` rather than being a "global" variable :)